gdk4/src/content_formats_builder.rs

```diff
@@ -0,0 +1,48 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{ContentFormats, ContentFormatsBuilder};
+use glib::translate::*;
+
+impl ContentFormatsBuilder {
+    #[doc(alias = "gdk_content_formats_builder_add_formats")]
+    pub fn add_formats(self, formats: &ContentFormats) -> Self {
```

Comment:
Rust builders usually don't use the `add_` prefix on functions. Also why are all these manual?

Reply:
If you drop the prefix, make sure to add another doc alias with the original function name `#[doc(alias ="add_formats")]`

Reply:
the functions actually add to a set of values, not a typical `member()` builder set

I made it manual mainly because I don't know how to tell gir to take self and -> Self. Is there a way?

Reply:
> the functions actually add to a set of values, not a typical member() builder set

Ah I guess, let's keep them as is then

> I made it manual mainly because I don't know how to tell gir to take self and -> Self. Is there a way?

No, i don't think so